rocminfo.cc

```diff
@@ -1185,6 +1185,70 @@ AcquireAndDisplayAgentInfo(hsa_agent_t agent, void* data) {
   return HSA_STATUS_SUCCESS;
 }
 
+static hsa_status_t
+showCPUInfo(hsa_agent_t agent, void* data) {
+  int pool_number = 0;
+  int isa_number = 0;
+
+  hsa_status_t err;
+  agent_info_t agent_i;
+
+  int *agent_number = reinterpret_cast<int*>(data);
+  (*agent_number)++;
+
+  err = AcquireAgentInfo(agent, &agent_i);
+  RET_IF_HSA_ERR(err);
+if(agent_i.device_type == HSA_DEVICE_TYPE_CPU){
+    DisplayAgentInfo(&agent_i);
+
+    printLabel("Pool Info:", true, 1);
+    err = hsa_amd_agent_iterate_memory_pools(agent, get_pool_info, &pool_number);
+    RET_IF_HSA_ERR(err);
+
+    printLabel("ISA Info:", true, 1);
+    err = hsa_agent_iterate_isas(agent, get_isa_info, &isa_number);
+    if (err == HSA_STATUS_ERROR_INVALID_AGENT) {
+      printLabel("N/A", true, 2);
+      return HSA_STATUS_SUCCESS;
+    }
+    RET_IF_HSA_ERR(err);
+}
+
+  return HSA_STATUS_SUCCESS;
+}
+
+static hsa_status_t
+showGPUInfo(hsa_agent_t agent, void* data) {
+  int pool_number = 0;
+  int isa_number = 0;
+
+  hsa_status_t err;
+  agent_info_t agent_i;
+
+  int *agent_number = reinterpret_cast<int*>(data);
```

Comment:
It seems agent_number is not used, and it can be removed

Reply:
Removed in 
https://github.com/ajitclarencejoseph/rocminfo/commit/026828a8a1e968c6121d81394fc4310d2f4757f8

Reply:
Looks good to me. Let me pull the patch in via our internal gerrit